packages/orbit-components/src/Table/index.tsx

```diff
@@ -144,20 +47,42 @@ const Table = ({
   }, [handleResize]);
 
   return (
-    <StyledTableOuter
-      ref={outer}
-      showShadows={shadows}
-      showLeft={left}
-      showRight={right}
+    <div
+      className={cx(
+        "relative w-full max-w-full",
+        "before:duration-normal before:bg-table-shadow-left before:absolute before:left-0 before:top-0 before:h-full before:w-[16px] before:transition-opacity before:ease-in-out",
+        !left && "before:opacity-0",
+        "after:duration-normal after:bg-table-shadow-right after:absolute after:right-0 after:top-0 after:h-full after:w-[16px] after:transition-opacity after:ease-in-out",
+        !right && "after:opacity-0",
+        shadows ? "before:block after:block" : "before:hidden after:hidden",
+      )}
       data-test={dataTest}
       id={id}
+      ref={outer}
     >
-      <StyledTableInner ref={inner} onScroll={handleScroll} showShadows={shadows}>
-        <StyledTable striped={striped} compact={compact} type={type} ref={table}>
+      <div
+        className={cx("w-full", shadows && "overflow-x-auto")}
+        onScroll={handleScroll}
+        ref={inner}
+      >
+        <table
+          className={cx(
```

Comment:
Just to confirm, we don't need `display: table`, right?

Reply:
yea, that's the default for `table`